### PR TITLE
feat: more emphasize to preferred login name in user's table and detail

### LINKED ISSUE #6555: Show "preferred login name" in user overview
### Preflight Checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe your problem

The user overview shows currently
- username
- email address

but not the "preferred login name", meaning the username + primary organization domain

Depending on the settings of the instance and the organizations users can have a different format of the "preferred login name" that they must use to authenticate. It is not guaranteed that this is either username or email address, but could be something different.

Screenshot 2023-09-14 at 12 05 20

It it hard to understand how the user can actually log in (what is the preferred login name) when looking at the overview. Also the detail view shows this info on the top/ side instead of more prominently on the user itself (eg, directly below the name or instead of "PROFILE").

Screenshot 2023-09-14 at 12 09 50

### Describe your ideal solution

As Manager, I want clearly see what "preferred login name" a user has to be able to help them login to the system. This is the primiary information about a user and should be shown in all user lists and also more prominently on the profile page.

### Version

_No response_

### Environment

ZITADEL Cloud

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by doncicuto**
As requested in #6555 this PR adds preferred login name as a new column to user list, removing the username column (human user) to keep the same number of columns for better readibility.

image

Also the preferred login name is added to the PROFILE title.

image

Acceptance criteria

- [X] Show "preferred login name" in users list
- [X] Give more emphasis to "preferred login name" in users profile

Closes #6555 

### Definition of Ready

- [X] I am happy with the code
- [X] Short description of the feature/issue is added in the pr description
- [X] PR is linked to the corresponding user story
- [X] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [X] No debug or dead code
- [X] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [X] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: console/src/app/pages/users/user-list/user-table/user-table.component.ts [hunk 1]
```diff
@@ -194,6 +194,9 @@ export class UserTableComponent implements OnInit {
         case 'username':
           sortingField = UserFieldName.USER_FIELD_NAME_USER_NAME;
           break;
+        case 'preferredLoginName':
+          sortingField = UserFieldName.USER_FIELD_NAME_USER_NAME;
```
Comment (hifabienne) [inline console/src/app/pages/users/user-list/user-table/user-table.component.ts:198]: I think this should be Preferred loginname instead of username?

Comment (hifabienne) [reply console/src/app/pages/users/user-list/user-table/user-table.component.ts:198]: i think that will do for the moment. @peintnermax what do you think?

Comment (peintnermax) [reply console/src/app/pages/users/user-list/user-table/user-table.component.ts:198]: I think this is fine for the moment.. The result rows might not be expected but the sorting could help 👍

**Comment by peintnermax** [comment]
@doncicuto could you remove the `USER.PROFILE.TITLE` from the i18n files? Its not needed anymore so I thinks its better to clean it up.. thx 👍

File: console/src/app/pages/users/user-detail/user-detail/user-detail.component.html [hunk 2]
```diff
@@ -78,7 +78,7 @@
 
         <ng-container *ngIf="currentSetting === 'general'">
           <ng-template cnslHasRole [hasRole]="['user.read$', 'user.read:' + user.id]">
-            <cnsl-card *ngIf="user.human" title="{{ 'USER.PROFILE.TITLE' | translate }}">
+            <cnsl-card *ngIf="user.human" title="{{ user.preferredLoginName }} - PROFILE">
```
Comment (peintnermax) [inline console/src/app/pages/users/user-detail/user-detail/user-detail.component.html:81]:
```suggestion
            <cnsl-card *ngIf="user.human" title="{{ user.preferredLoginName }} - {{ 'USER.PROFILE.TITLE' | translate }}">
```

**Review by peintnermax — CHANGES_REQUESTED** [review]

File: console/src/app/pages/users/user-list/user-table/user-table.component.ts (hunk 1 above)
Comment (peintnermax) [inline console/src/app/pages/users/user-list/user-table/user-table.component.ts:198]:
```suggestion
          // TODO: replace with preferred username sorting once implemented
          sortingField = UserFieldName.USER_FIELD_NAME_USER_NAME;
```

File: console/src/app/pages/users/user-detail/user-detail/user-detail.component.html (hunk 2 above)
Comment (doncicuto) [reply console/src/app/pages/users/user-detail/user-detail/user-detail.component.html]: Done! thanks as always @peintnermax

File: console/src/app/pages/users/user-list/user-table/user-table.component.ts (hunk 1 above)
Comment (doncicuto) [reply console/src/app/pages/users/user-list/user-table/user-table.component.ts:199]: Hope I can fix this TODO in the future. Thanks @peintnermax

**Review by peintnermax — APPROVED** [review]

**Pull request merged**